practice/tic-tac-toe/dict_board.py

```diff
@@ -62,6 +71,67 @@ def place_token(self, x, y, token):
         """
         self._coord_to_token[(x, y)] = token
 
+    def make_rows(self):
+        """Returns a list of listed tokens, in order of the rows going top down.
+
+        >>> a = DictTTTBoard()
+        >>> a._coord_to_token = {
+        ...   (0,0): 'X', (1,0): ' ', (2,0): 'O',
+        ...   (0,1): 'X', (1,1): 'O', (2,1): 'X',
+        ...   (0,2): 'O', (1,2): ' ', (2,2): 'X'
+        ...    }
+        >>> a.make_rows()
+        [['X', ' ', 'O'], ['X', 'O', 'X'], ['O', ' ', 'X']]
+        """
+        coords_to_tokens = sorted(list(self._coord_to_token.items()))
+        row_1 = [coord_to_token[1] for coord_to_token in coords_to_tokens
+                 if coord_to_token[0][1] == 0]
+        row_2 = [coord_to_token[1] for coord_to_token in coords_to_tokens
+                 if coord_to_token[0][1] == 1]
+        row_3 = [coord_to_token[1] for coord_to_token in coords_to_tokens
+                 if coord_to_token[0][1] == 2]
+        return [row_1, row_2, row_3]
```

Comment:
There's a pretty easy pattern here that could be made more concise with a for loop and `range(3)`.